scenarioo-server/src/main/java/org/scenarioo/rest/configuration/ConfigurationResource.java

```diff
@@ -41,7 +42,16 @@
 	public Configuration getConfiguration() {
 		return configurationRepository.getConfiguration();
 	}
-	
+
+	@GET
+	@Path("/diffViewer")
+	@Produces({ "application/json", "application/xml" })
+	public DiffViewerStatus getDiffViewerStatus() {
```

Comment:
@danielsuter What i mean: this service should return a new class called ScenariooStatus, and this status should contain all kind of stati that is needed in config page (I would even include the Configuration object as well), so we just have to call one service for everything needed on this config page.

The service would be called "getScenariooStatus" then and it will also return the search engine status in the same call.